lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -142,6 +145,15 @@ export abstract class ClientApplication {
             new BrowserCacheManager(this.config.auth.clientId, this.config.cache, this.browserCrypto, this.logger) :
             DEFAULT_BROWSER_CACHE_MANAGER(this.config.auth.clientId, this.logger);
 
+        if (this.config.system.allowNativeBroker) {
+            const nativeCacheOptions: Required<CacheOptions> = {
+                cacheLocation: BrowserCacheLocation.MemoryStorage,
+                storeAuthStateInCookie: false,
+                secureCookies: false
+            };
+            this.nativeInternalStorage = new BrowserCacheManager(this.config.auth.clientId, nativeCacheOptions, this.browserCrypto, this.logger);
```

Comment:
Likewise, we can probably initialize this for all scenarios so we don't need to worry about `undefined`